doc/classes/ResourceLoader.xml

```diff
@@ -47,9 +47,9 @@
 				Returns the dependencies for the resource at the given [param path].
 				[b]Note:[/b] The dependencies are returned with slices separated by [code]::[/code]. You can use [method String.get_slice] to get their components.
 				[codeblock]
-				for dep in ResourceLoader.get_dependencies(path):
-				    print(dep.get_slice("::", 0)) # Prints UID.
-				    print(dep.get_slice("::", 2)) # Prints path.
+				for dependency in ResourceLoader.get_dependencies(path):
+				    print(dependency.get_slice("::", 0)) # Prints the UID.
+				    print(dependency.get_slice("::", 2)) # Prints the path.
```

Comment:
Are we starting to migrate GDScript examples to using fully spelled-out names like this? I'm not opposed, but this seems to somewhat break with existing usage.

Currently, it seems that short abbreviated names for variables are common and idiomatic for GDScript (unlike C#, where being verbose is favored). "dep" here seems like a very natural name. I'm not sure if this was ever intentionally decided, or just a byproduct of the example writers being C++ programmers 😛

Reply:
> Currently, it seems that short abbreviated names for variables are common and idiomatic for GDScript (unlike C#, where being verbose is favored).

How? I don't feel like that's the case. We have a tendency to discourage abbreviations, sometimes even in the internal code.

> I'm not sure if this was ever intentionally decided, or just a byproduct of the example writers being C++ programmers 😛

No decision. A lot of the class reference is written from the perspective of the developer that came up with it -- or perhaps Reduz alone -- so it comes off as "skimpy" and "technological" in ways such as this.

Examples need to be understandable first and foremost, and in this specific case the difference between "dep" and "dependency" is not too drastic.
If anything, the variable's name could be changed to imply it's a String because I can totally see some possible confusion here.

Reply:
Yeah, I'm looking again at all the code examples and abbreviations are a lot less common than my initial impression, only in 20% or less of the examples. I think I just got the impression it was widespread and intentional, because I was comparing to C#.

I'm all in favor of changing this one to "dependency".